tests/js/support/testConfig.js

```diff
@@ -21,7 +21,24 @@
 
 // If the docker instance has imported this stitch config, it will have written the app id
 // back into the config file, so we can read it out again here.
-const integrationTestsAppId = `${require("../../../src/object-store/tests/mongodb/stitch.json").app_id}`;
+
+// Prevent React Native packager from seeing modules required with this
+const require_method = require;
+function nodeRequire(module) {
+    return require_method(module);
+}
+
+let pathToStitchJson = "../../../src/object-store/tests/mongodb/stitch.json";
+const isNodeProcess = typeof process === 'object' && process + '' === '[object process]';
+
+if (isNodeProcess && process.env.ELECTRON_TESTS_REALM_MODULE_PATH) {
+    const path = nodeRequire("path");
+    console.log("ELECTRON_TESTS_REALM_MODULE_PATH " + process.env.ELECTRON_TESTS_REALM_MODULE_PATH);
+    pathToStitchJson = path.resolve(process.env.ELECTRON_TESTS_REALM_MODULE_PATH, '../../../../src/object-store/tests/mongodb/stitch.json')
+}
+console.log("pathToStitchJson " + pathToStitchJson);
+
+const integrationTestsAppId = `${nodeRequire(pathToStitchJson).app_id}`;
```

Comment:
Not big fan og calling functions from a string template (as errors thrown are a bit less obvious to debug from the stacktrace). Also the string template seems unnecessary.

```suggestion
const stitchConfiguration = nodeRequire(pathToStitchJson);
const integrationTestsAppId = stitchConfiguration.app_id;
```

Reply:
not my change. see previous version